### PR TITLE
chore: readme platform libraries

### PR DESCRIPTION
**Pull request by ThyMinimalDev**
## What does this PR do?

README of platform libraries go explain how to work with it in dev

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

File: packages/platform/libraries/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+since the versions are matching in both package.json yarn will try to use the locally built code
+
+now go to packages/platform/libraries and do
+
+- yarn build
+
+then go back to /apps/api/v2 and run
+
+- yarn
+
```
Comment (supalarry) [inline packages/platform/libraries/README.md:31]: For dev I would also add to not commit version changes of platform-libraries, or maybe we can just always leave it to 0.0.0 for development and remove part of docs changing platform-libraries version locally, and only change v2 by adding a pointer to 0.0.0.

File: packages/platform/libraries/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- yarn dev
+
+# Before Merging to main
+- get your version of libraries published on NPM
```
Comment (supalarry) [inline packages/platform/libraries/README.md:35]: I would add more info about publishing - To be able to publish you must be a contributor to the platform libraries npm package, and then have to npm auth yourself via the CLI. Then, make sure to check latest platform libraries version at npm and then increase version number. Run `yarn publish`.

Comment (ThyMinimalDev) [reply packages/platform/libraries/README.md:35]: updating this

**Review by supalarry — APPROVED** [review]

File: packages/platform/libraries/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- yarn dev
+
+# Before Merging to main
+- Publish Your Version of Libraries on NPM:
+	- To publish, ensure you are a contributor to the platform libraries' NPM package.
+	- Authenticate yourself via the CLI using npm auth.
+	- Increment the version number accordingly.
+	- Run yarn publish to publish your version.
+- Once it's published, change back the version in packages/platform/libraries/package.json back to 0.0.0
```
Comment (supalarry) [inline packages/platform/libraries/README.md:40]: And also need to update imports from latest platform libraries version instead of local v1.2.3

**Pull request merged**